roundup/github.py

```diff
@@ -18,7 +18,7 @@ def compare_digest(a, b):
 URL_RE = re.compile(r'https://github.com/python/cpython/pull/(?P<number>\d+)')
 VERBS = r'(?:\b(?P<verb>close[sd]?|closing|fix(?:e[sd])?|)\s+)?'
 ISSUE_RE = re.compile(r'%sbpo-(?P<issue_id>\d+)' % VERBS, re.I|re.U)
-BRANCH_RE = re.compile(r'(2\.\d|3\.\d|master)', re.I)
+BRANCH_RE = re.compile(r'(3\.\d+|main)', re.I)
```

Comment:
test/data/pushevent.txt and test/data/pushevent1.txt still contains ``"ref":"refs/heads/master"``. Either remove the test, update the test, or still accept "master".

Also, test/data/pushevent2.txt contains ``"ref":"refs/heads/2.7" ``.

Reply:
I can update tests once we deploy this to prod.